dagster/helm/dagster/values.yaml

```diff
@@ -51,7 +51,7 @@ dagster:
   pipelineRun:
     image:
       repository: dkr.plural.sh/dagster/dagster/user-code-example
-      tag: 1.2.7
+      tag: 1.2.7-extended
```

Comment:
do we need this pipeline run resource actually?

Reply:
No we don't